readme.txt

```diff
@@ -36,19 +43,9 @@ Follow along with or [contribute](https://github.com/Automattic/amp-wp/blob/deve
 1. Article from TNW showing customized AMP template.
 1. Article from Halfbrick showing customized AMP template.
 
-== Installation ==
```

Comment:
There is a benefit to having a dedicated "Installation" section named as such: it will be shown in a separate tab, like so: https://wordpress.org/plugins/jetpack/#installation